Annex60/Experimental/Benchmarks/AirFlow/Components/Floor.mo

```diff
@@ -0,0 +1,173 @@
+within Annex60.Experimental.Benchmarks.AirFlow.Components;
+model Floor
+  "Floor element for air flow benchmark, consisting of zones, hallway, outdoor environment, and staircase"
+
+  parameter Integer nZones(min=1) = 4 "Number of zone elements";
+
+  parameter Modelica.SIunits.Temperature TRoom = 298.15
+    "Indoor air temperature of room in K";
+  parameter Modelica.SIunits.Temperature THallway = 293.15
+    "Indoor air temperature of hallway in K";
+  parameter Modelica.SIunits.Temperature TStaircase = 293.15
+    "Indoor air temperature of staircase in K";
+  parameter Modelica.SIunits.Height heightRooms = 3 "Height of rooms in m";
+
+  parameter Modelica.SIunits.Length lengthZone = 5 "Length of room in m";
+  parameter Modelica.SIunits.Length widthZone = 5 "Width of room in m";
+  parameter Modelica.SIunits.Length widthHallway = 3 "Width of room in m";
+  parameter Real doorOpening = 1
+    "Opening of door (between 0:closed and 1:open)";
+
+  replaceable package Medium = Modelica.Media.Air.SimpleAir;
```

Comment:
This should be a replaceable partial package so that it must be assigned from the top-level model.
Also, we should use `Annex60.Media.Air` in the benchmarks.